test/harvest/test_pubmed.py

```diff
@@ -44,11 +47,85 @@ def existing_publication(test_session):
         return pub
 
 
+def mock_jsonl(path):
```

Comment:
Does this actually need to be so detailed?

Reply:
i removed some data but wanted it to be at least somewhat similar to the json structure of an actual record